src/components/inline/Button/Button.js

```diff
@@ -4,22 +4,28 @@ import React from "react";
 
 import s from "./Button.module.css";
 
-const Button = ({ text, externalLink, internalLink, onClick }) => {
+const Button = ({ text, externalLink, internalLink, onClick, noHover }) => {
+  const button = `${s.button} ${noHover} ? ${s.noHover}: ""}`;
+
   if (externalLink) {
     return (
       <a
         rel="noreferrer"
         target="_blank"
         href={externalLink}
-        className={s.button}
+        className={button}
       >
         {text}
       </a>
     );
   }
   if (internalLink) {
     return (
-      <Link to={internalLink} className={s.button}>
+      <Link
+        className={button}
+        style={noHover ? { top: "0" } : {}}
```

Comment:
Can you remove this line? This isn't needed with your conditional class name anymore.

Reply:
you mean line 26? You labeled 22